Codeinwp/otter-blocks
# [Rock] Add PHP unit tests

Closes #1623


### Summary


Added units tests for:
- Dynamic Content query parse and small evaluation test
- Dynamic Conditions evaluation
- Stripe_API functions unit test
- Pro Patterns fetching test

Locally the test can be run with Docker provided by Gutenberg packages.
```bash
npm run test:unit:php:setup # start the container
npm run test:unit:php:base # run the tests from `./tests` folder
```


### Test instructions


1. A slight change was made in Dynamic Content. Have a test instance with the current version, and after switching with the one from this PR, check if it still works.
2. Same with Stripe -> check Stripe Checkout Block


---- 

### Checklist before the final review
